packages/mrm-task-package/index.js

```diff
@@ -71,9 +91,14 @@ module.exports.parameters = {
 		message: 'Enter minimum supported Node.js version',
 		default: 10,
 	},
+	version: {
+		type: 'input',
+		message: 'Enter project version',
+		default: () => init().version || '1.0.0',
```

Comment:
This sound more like an npm config than `init()` (which makes an impression that it's actually initializing a package as `npm init` does).

Reply:
Since the file I'm getting it from is `npm` and it exports `install`/`uninstall`, I guess `config` could be more consistent than `npmConfig`, so I've gone with that for now, but let me know.

Reply:
Guess I could probably should have said `getConfig`, but let me know. I'm if anything too flexible with names. :-)

Reply:
I went with `getConfig` on the latest, as well as removing from `mrm-core` and rebasing directly onto `master`. If you do find some time to gradually get to the testing PRs, though, I'd be most grateful, as I'd feel more confident building additional PRs on top. Thanks!